docs/dev/build-instructions/build_linux.md

```diff
@@ -31,6 +31,32 @@ sudo apt install \
   g++-multilib
 ```
 
+In Cent OS 8 you need to use
+```bash
+sudo dnf -y install epel-release
+sudo dnf config-manager --set-enabled PowerTools
+sudo dnf group install "Development Tools"
+sudo dnf install https://zeroc.com/download/ice/3.7/el8/ice-repo-3.7.el8.noarch.rpm
+sudo dnf install libice-c++-devel libice3.7-c++
+sudo dnf install \
+qt-creator \
```

Comment:
Are you sure that you need Qt Creator? Shouldn't that only be the IDE that is per se unrelated to building the project?

Reply:
Thanks for your kind feedback. Yes, expect qt-creator can be removed, will check again, this was needed before transitioning to CMake. A separate file for dependencies for different distributions with Cent OS as a section is also a good suggestion. There is also some material for the wiki that could be incorporated/linked if a separate page is used:
https://wiki.mumble.info/wiki/Install_CentOS7

Link to cmake page is because it is easier to build from source for Cent OS 8 - may also apply to other distributions. Will ake a sepaate pull request for this.

Reply:
A separate PR wouldn't have been necessary. It's fine having multiple commits for multiple things in a single PR. The point being only that a single commit should only be for a single change. But it doesn't hurt this way either :)

The wiki page you linked is about installing and setting up instead of compiling though. Ultimately this is also something that we want to document within this repo, but this would be a different doc-tree than the `build-instructions` one :point_up: 

> A separate file for dependencies for different distributions with Cent OS as a section is also a good suggestion.

Alright, then let's use that :+1:

Reply:
qt-creator is not needed.  libcap-devel should replace libcap-ng-devel

It then builds.